pyproject.toml

```diff
@@ -7,23 +7,23 @@ authors = ["Torchbox Ltd"]
 [tool.poetry.dependencies]
 python = "^3.11"
 Django = "~4.2"
-wagtail = "~6.1"
+wagtail = "~6.2"
 psycopg2 = "^2.9.9"
-gunicorn = {version = "^22.0.0", optional = true}
-whitenoise = "^6.1.0"
-phonenumbers = "^8.13.38"
+gunicorn = {version = "^23.0.0", optional = true}
+whitenoise = "^6.7.0"
+phonenumbers = "^8.13.47"
 Wand = "^0.6.10"
 
 # Django packages
-django-basic-auth-ip-whitelist = "^0.5"
+django-basic-auth-ip-whitelist = "^0.6"
```

Comment:
[v0.6 introduces some breaking changes for IP detection](https://github.com/torchbox/django-basic-auth-ip-whitelist/releases/tag/v0.6.0).

Please see https://github.com/Buckinghamshire-Digital-Service/buckinghamshire-council/commit/2b9c927ae0d9641d0896ddb69da75b63a34e9b29 for an example of how to handle this

Reply:
Thanks! I've read the release notes and the readme for `django-xff`. I've set it up in the same way as Nick did in the commit you've linked, but I'm honestly not sure if those are the correct settings for this project. Do you know how to identify a correct proxy depth for example?

f734a92e6309fa3564d4a8ff7fab30b5a2024f7b

Reply:
Thanks, unfortunately I'm not very certain regarding the basis for correct determination of proxy depth. Thanks for raising the question in Slack, with the responses from the team, I now have a better understanding!